wagtail/wagtailsearch/signal_handlers.py

```diff
@@ -22,16 +26,24 @@ def post_save_signal_handler(instance, **kwargs):
     indexed_instance = get_indexed_instance(instance)
 
     if indexed_instance:
-        for backend in get_search_backends(with_auto_update=True):
-            backend.add(indexed_instance)
+        for backend_name, backend in get_search_backends_with_name(with_auto_update=True):
+            try:
+                backend.add(indexed_instance)
+            except:
```

Comment:
Thanks for the suggestion! I'll switch it to use ``Exception``.

I think it would be good to use the original exception as we log them. We might lose useful information if we rely on the developer translating them to ``SearchBackendException``